tests/Behat/Gherkin/ParserExceptionsTest.php

```diff
@@ -61,22 +61,19 @@ public function testTextInBackground()
     {
         $feature = <<<GHERKIN
 Feature: Behat bug test
-    Background: remove X to couse bug
-    Step is red form is not valid
-    asd
-    asd
-    as
-    da
-    sd
-    as
-    das
-    d
```

Comment:
this is changing what is tested (and might not prevent regressions anymore)

Reply:
That was part of PR #173 - I have adjusted that PR now so that it does not touch the data (feature//background) to be tested.

This PR #174 will need to be rebased anyway after those other PRs are finalized and merged.